content/faq/bookmarks/how-to-import/index.ru.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Как импортировать метки?
+description: ""
+updated: "2024-06-12"
+
+taxonomies:
+  faq: ["Метки и Треки"]
+
+extra:
+  order: 20
+---
+
+Вы можете импортировать метки отправленные из Organic Maps или сторонних приложений, если они позволяют экспортировать метки в форматах KML, KMZ, GPX:
```

Comment:
```suggestion
Вы можете импортировать метки, отправленные из Organic Maps или сторонних приложений в форматах KML, KMZ, KMB, GPX:
```

Reply:
Let's mass-replace the string in all translations and add KMB everywhere.

Reply:
Added KMB to the list of supported formats all over the FAQ.